clients/venice-admin-tool/src/main/java/com/linkedin/venice/datarecovery/StoreRepushCommand.java

```diff
@@ -0,0 +1,366 @@
+      this.password = password;
+    }
+
+    public String getPassword() {
+      return this.password;
+    }
+
+    public void setDebug(boolean debug) {
+      this.debug = debug;
+    }
+
+    public void setCommand(String cmd) {
+      this.command = cmd;
+    }
+  }
+
+  public static class Result {
+    private String cluster;
+    private String store;
+    private String stdOut;
+    private int exitCode;
+    private String error;
+    private String message;
+
+    public int getExitCode() {
+      return exitCode;
+    }
+
+    public void setExitCode(int exitCode) {
+      this.exitCode = exitCode;
+    }
+
+    public String getStdOut() {
+      return stdOut;
+    }
+
+    public void setStdOut(String stdOut) {
+      this.stdOut = stdOut;
+    }
+
+    public boolean isError() {
+      return error != null;
+    }
+
+    public String getCluster() {
+      return cluster;
+    }
+
+    public void setCluster(String cluster) {
+      this.cluster = cluster;
+    }
+
+    public String getStore() {
+      return store;
+    }
+
+    public void setStore(String store) {
+      this.store = store;
+    }
+
+    public void setError(String error) {
+      this.error = error;
+    }
+
+    public String getError() {
+      return error;
+    }
+
+    public String getMessage() {
+      return message;
+    }
+
+    public void setMessage(String message) {
+      this.message = message;
+    }
+
+    public void parseStandardOutput() {
+      // No standard output or empty output.
+      if (stdOut == null || stdOut.equals(StringUtils.EMPTY)) {
+        return;
+      }
+
+      // Command reached to Azkaban, no matter it was a success or a failure.
+      if (matchAzkabanSuccessPattern()) {
+        return;
+      }
+
+      if (matchAzkabanFailurePattern()) {
+        return;
+      }
+
+      // Failed: repush command itself hit an error (e.g. incomplete parameters)
+      error = stdOut;
+    }
+
+    private boolean matchAzkabanSuccessPattern() {
+      // Success: (example) https://example.com/executor?execid=21585379
+      String successPattern = "^https(.*)execid(.*)$";
+      Pattern pattern = Pattern.compile(successPattern, Pattern.CASE_INSENSITIVE | Pattern.MULTILINE);
+      Matcher matcher = pattern.matcher(stdOut);
+      if (matcher.find()) {
+        message = matcher.group();
+        return true;
+      }
+      return false;
+    }
+
+    private boolean matchAzkabanFailurePattern() {
+      // Failed: Handle azkaban failed response to check if the output line contains any errors. (e.g. invalid password)
+      String errorPattern = "^(.*)Response(.*)error(.*)$";
+      Pattern pattern = Pattern.compile(errorPattern, Pattern.CASE_INSENSITIVE | Pattern.MULTILINE);
+      Matcher matcher = pattern.matcher(stdOut);
+      // Find the first occurrence of an error line and report.
+      if (matcher.find()) {
+        error = matcher.group();
+        return true;
+      }
+      return false;
+    }
```

Comment:
These are internal implementations. We should not tie OSS Venice to Azkaban/`venice-tools` response patterns